homeassistant/components/youless/sensor.py

```diff
@@ -212,6 +212,38 @@ class YouLessSensorEntityDescription(SensorEntityDescription):
             lambda device: device.phase3.current.value if device.phase1 else None
         ),
     ),
+    YouLessSensorEntityDescription(
+        key="tariff",
+        device_group="power",
+        translation_key="active_tariff",
+        device_class=SensorDeviceClass.ENUM,
+        options=["1", "2"],
```

Comment:
How does this transalte to what this mean? Is it common to have 1 or 2? or is it high and low?

Reply:
The official documentation recognizes only tariff 1 and 2, which is also why the sensor translations were updated in the previous PR to reflect that.

Reply:
Looking at the screenshot of the issue #135737 it appears the app also uses the tariff number in the labels, rather then high and low. So in that sense the labels in the integration used to be wrong.

Reply:
In that case I think this is fine